kubernetes/ansible/roles/secor-deploy/defaults/main.yaml

```diff
@@ -5,6 +5,10 @@ image_pullPolicy: IfNotPresent
 imagepullsecrets: sunbird-registry-secret
 secor_storage_class: managed-premium
 
+# this varible is to control the monitoring deployment. override this value by adding into
+# private repo common.yml to not to create the secor lag alert rules.
+alertrule_enabled: true
```

Comment:
I think you should prepend secor_ else it'll be a common name, and won't make sense in private file

Reply:
Updated the var name @rjshrjndrn